modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/stat/IgniteStatisticsManagerImpl.java

```diff
@@ -260,12 +289,14 @@ public IgniteStatisticsRepository statisticsRepository() {
 
     /** {@inheritDoc} */
     @Override public void dropAll() throws IgniteCheckedException {
+        checkStatisticsState("drop all statistics");
+
         statCfgMgr.dropAll();
     }
 
     /** {@inheritDoc} */
     @Override public void stop() {
-        disableOperations();
+        stateChanged();
```

Comment:
StatChanged callback shouldn't called here. A callback should call this (stop) method instead.

Reply:
Done.